runtime/client_http_request.go

```diff
@@ -82,7 +85,7 @@ func (req *ClientHTTPRequest) start(
 
 // WriteJSON will send a json http request out.
 func (req *ClientHTTPRequest) WriteJSON(
```

Comment:
Consider renaming method to WriteJSONReq or WriteHTTPJSON to make it clearer that it's the whole request with headers and not just the json body.